example/src/SignUp.js

```diff
@@ -13,15 +12,21 @@ class SignUp extends Component {
     }
     handleChange(event) {
         this.setState({
-          [event.target.name]: event.target.value
+            [event.target.name]: event.target.value
```

Comment:
Why the indent?

Reply:
I think I pressed the wrong button in my code editor. I'll make sure to check with `git diff`.